packages/strapi-plugin-versioning/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "strapi-plugin-versioning",
+  "version": "0.1.0",
+  "description": "Strapi plugin for collecting versions of content type entities",
+  "main": "src/index.ts",
```

Comment:
for plugin as I know You souldn;t specify index.ts - I didn't even notice it in this PR

Reply:
removed